packages/playwright/src/index.ts

```diff
@@ -259,6 +270,9 @@ export default class AxeBuilder {
         partialResults,
         options
       })
+      .catch(e => {
+        return e;
```

Comment:
Why `return` the error? Doesn't this mean it's "dropped"?

Reply:
This means the Promise resolves with an instance of an `Error`, thus breaking _all_ type contracts.

```
var result = await new Promise((resolve, reject) => reject(new Error('boom'))).catch(error => { return error })
result instanceof Error
// => true
result
// => Error: boom
    at result (<anonymous>:1:60)
    at new Promise (<anonymous>)
    at <anonymous>:1:20
```